scripts/centralize_docs.py

```diff
@@ -0,0 +1,180 @@
+#! /usr/bin/env python3
+
+import pathlib
+import json
+import tempfile
+import subprocess
+import sys
+from collections import defaultdict
+from copy import deepcopy
+
+
+def _load_jsonl(file):
+    with open(file, 'r') as f:
+        _d = [json.loads(line) for line in f]
+        try:
+            data = {int(doc['metadata']['pmcid']): doc for doc in _d}
```

Comment:
I think this is probably because @koudyk  annotated some abstracts that don't have a pmcid?

Reply:
Correct. The plan would be to allow that still by keeping those in `{project_name}/documents`. So that would be reserved for documents not handled by pubget.

Could handle that differently.